lib/sonice/public/script.js

```diff
@@ -116,6 +116,19 @@ $(function() {
     })
   }
 
+  // Toggle the playpause button on the remote button widget
+  var togglePlayPause = function(toggle_playpause_button){
```

Comment:
@awongCM : houndci is an automated tool that tries to enforce a common style. Don't pay too much attention to what it says since it wasn't there from the start of this project (hound recently added JavaScript, too).

I would be better if it was `togglePlayPauseButton` but I'll gladly pull this is as it is, I won't be very fussy about this code style for this project.